raft/raft.go

```diff
@@ -230,7 +230,7 @@ func newRaft(c *Config) *raft {
 		nodesStrs = append(nodesStrs, fmt.Sprintf("%x", n))
 	}
 
-	r.logger.Infof("newRaft %x [peers: [%s], term: %d, commit: %d, applied: %d, lastindex: %d, lastterm: %d]",
+	r.logger.Infof("newRaft %d [peers: [%s], term: %d, commit: %d, applied: %d, lastindex: %d, lastterm: %d]",
```

Comment:
We want to keep it compact and consistent with other ID print out format inside etcd repo (yea, we want to keep raft a separate thing though. If people love dec, we can make it dec). 

Actually here we should print out as %16x rather than %x probably. Every bits in the ID should be printed out.